### ISSUE TITLE
Question: Can gif looping be disabled from agg?

### ISSUE THREAD

**Issue by gerhard** [asciinema/agg#13]
I generate the gif with `agg`, then disable looping with `gifsicle`.

Is it possible to disable looping from `agg`, without running `gifsicle` afterwards?

Have:

```sh
agg cool.cast cool-loop.gif
gifsicle --no-loopcount cool-loop.gif > cool.gif
```

Want:

```
agg --no-loop cool.cast cool.gif
```

Thank you!

**Comment by sickill** [comment]
Yeah, it looks like gifski allows controlling looping:

https://github.com/ImageOptim/gifski/blob/5f571d9a3315a0450c285e69b3047043e0612a72/src/lib.rs#L55-L58

So we could add a flag for that.

**Comment by sickill** [comment]
Just released v1.3 with new `--no-loop` flag.
